### DISCUSSION TITLE
How many locations/logbooks do you have configured?

### DISCUSSION THREAD

**Discussion by ei8fdb** [magicbug/Cloudlog#3219] [Polls]
To help with the UI design [I've mentioned](https://github.com/magicbug/Cloudlog/discussions/3217) for quickly switching logbook/locations, it'd be helpful to know how many locations people have configured in Cloudlog.

Github allows only 1 poll in a discussion, so I'd appreciate if you could also leave a comment saying how many logbooks you've configured.

**Comment by ei8fdb** [comment]
About 7 logbooks configured

**Reply by magicbug** [reply]
10 logbooks, 50 odd locations :)

**Reply by Craig3FW** [reply]
> About 7 logbooks configured

I've around 12 Logbooks .... EI3FW ... F/EI3FW/P .... MW/EI3FW/P and so on, one for each DXCC except EI where I have the home and portable. I then have well over 50 locations.

An easy reminder would be great, I've forgotten many times.

**Comment by pa3hfj** [comment]
I am running 2 instances of Cloudlog; one for my own logs and one for our local hamradioclub department. For myself I use one logbook with almost 30 locations (due to /P holidays etc.) and for the club we are using about 12 logbooks (all with separate logins per special callsign) with each normally one or sometimes several locations.

By the way, what I still do miss is the possibility to use several users on the same logbook. We could do that in the earlier versions of Cloudlog and was very handy for the special events where the several operators could login under there own user-account. That makes it impossible to differentiate the QSO's per operator and is also from security perspective not so good. But this is about the only remark I could give on this very fine piece of software, I do love it every single day I use it !!
